modules/dnn/src/layers/cpu_kernels/convolution.hpp

```diff
@@ -89,6 +94,7 @@ struct FastConv
     float16_t* weightsWinoBufPtr_FP16;
 #endif
 
+    Ptr<WinoParams> winoParams = makePtr<WinoParams>();
```

Comment:
Ptr to structure with constants is overkill for me. I propose do the following:
- remain const int for NEON and other non-Intel platforms.
- for x86 branch use something like `static const int CONV_WINO_IBLOCK = (checkHardwareSupport(CPU_AVX) || checkHardwareSupport(CPU_AVX2)) 6 : 3;`
It means that:
- no code changes in compute part
- no performance side effects for others
- single instance of constant values, but not copy for each layer

Also `useAVX`, `useAVX2` and others may be `static const` too.